ops/docker-compose-dev.yml

```diff
@@ -1,7 +1,8 @@
 version: '2'
 services:
-  db:
-    build: docker/images/mysql
+  mysql:
+    container_name: mysql
```

Comment:
why do we need `container_name`?

Reply:
Use `docker-compose` - will resolve all of your problems with container naming...